tests/em/fdem/forward/test_FDEM_analytic_Dipoles.py

```diff
@@ -0,0 +1,1494 @@
+import unittest
+from SimPEG import EM, Mesh, Utils, np, Maps, sp
+try:
+    from pymatsolver import MumpsSolver
+    solver = MumpsSolver
+except ImportError:
+    from SimPEG import SolverLU
+    solver = SolverLU
+# import sys
+from scipy.constants import mu_0
+
+plotIt = False
+
+if plotIt:
+    import matplotlib.pylab
+    import matplotlib.pyplot as plt
+
+class FDEM_analytic_DipoleTests_CylMesh(unittest.TestCase):
+
+    def setUp(self):
+
+        # Define model parameters
+        self.sigmaback = 1.
+        # mu = mu_0*(1+kappa)
+        self.kappa = 1.
+
+        # Set source parameters
+        self.freq = 1.
+        self.src_loc = np.r_[0., 0., 0.]
+
+        # Compute skin depth
+        skdpth = 500. / np.sqrt(self.sigmaback * self.freq)
+
+        # Create cylindrical mesh
+        self.csx, self.ncx, self.npadx = 5, 50, 25
```

Comment:
this is a relatively big cyl mesh 100 x 100 cells. Can we reduce the size so that the tests run more quickly?